crates/pubnub_transport_reqwest/src/reqwest.rs

```diff
@@ -81,6 +80,9 @@ fn prepare_path(path: String, query_params: HashMap<String, String>) -> String {
 
 #[cfg(test)]
 mod should {
+    fn init() {
+        env_logger::init();
+    }
```

Comment:
tbh do we need that logging in the tests?
Unit tests probably will exactly tell you what's wrong so imo we should make small UT to avoiding confusion. 

It would be useful in the contract tests when we will check every part of the SDK if that works as expected.